code/modules/admin/verbs/randomverbs.dm

```diff
@@ -781,7 +786,18 @@ Traitors and the like can also be revived with the previous role mostly intact.
 	if(shuttle_master.emergency.mode >= SHUTTLE_DOCKED)
 		return
 
-	shuttle_master.emergency.cancel()
+	if(shuttle_master.emergency.canRecall == FALSE)
+		if(alert("Shuttle is currently set to be nonrecallable. Recalling may break things. Respect Recall Status?", "Override Recall Status?", "Yes", "No") == "Yes")

```

Comment:
Isn't this going to fail/do nothing when the admin selects 'Yes' to proceed?

Reply:
Correct, if you just spam click, it will default to yes, thus killing the recall prompt. It may be unexpected so it's safer to have the default kill the action rather than continue. Additionally it would be even stranger to flip the order of yes/no buttons. Obviously, I'm open to suggestions on wording and general implementation, but that's just going off of what @tigercat2000 already suggested.